judo-runtime-core-jsl-itest/common/src/main/java/hu/blackbelt/judo/runtime/core/jsl/fixture/JudoRuntimeFixture.java

```diff
@@ -0,0 +1,201 @@
+package hu.blackbelt.judo.runtime.core.jsl.fixture;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.Module;
+import hu.blackbelt.judo.meta.expression.builder.jql.JqlExpressionBuilderConfig;
+import hu.blackbelt.judo.meta.expression.builder.jql.asm.AsmJqlExtractor;
+import hu.blackbelt.judo.runtime.core.bootstrap.JudoDefaultModule;
+import hu.blackbelt.judo.runtime.core.bootstrap.JudoModelLoader;
+import hu.blackbelt.judo.runtime.core.bootstrap.dao.rdbms.hsqldb.JudoHsqldbModules;
+import hu.blackbelt.judo.runtime.core.bootstrap.dao.rdbms.postgresql.JudoPostgresqlModules;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.hsqldb.HsqldbDialect;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.hsqldb.HsqldbRdbmsInit;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.liquibase.SimpleLiquibaseExecutor;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.postgresql.PostgresqlDialect;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.postgresql.PostgresqlRdbmsInit;
+import hu.blackbelt.judo.runtime.core.query.QueryFactory;
+import hu.blackbelt.mapper.api.ExtendableCoercer;
+import hu.blackbelt.mapper.impl.DefaultCoercer;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.emf.common.util.ECollections;
+import org.eclipse.emf.common.util.URI;
+import org.slf4j.bridge.SLF4JBridgeHandler;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.TransactionStatus;
+import org.springframework.transaction.support.DefaultTransactionDefinition;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import uk.org.lidalia.sysoutslf4j.context.SysOutOverSLF4J;
+
+import java.io.File;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static hu.blackbelt.judo.runtime.core.jsl.fixture.JudoDatasourceFixture.DIALECT_HSQLDB;
+import static hu.blackbelt.judo.runtime.core.jsl.fixture.JudoDatasourceFixture.DIALECT_POSTGRESQL;
+import static java.util.Objects.requireNonNullElse;
+
+@Slf4j
+public class JudoRuntimeFixture {
+
+    static {
+        SLF4JBridgeHandler.install();
+        SysOutOverSLF4J.sendSystemOutAndErrToSLF4J();
+    }
+
+    public static final String MODEL_SOURCES = "target/generated-test-sources/model";
+
+    Injector injector;
+
+    private TransactionStatus transactionStatus;
+
+    private PlatformTransactionManager transactionManager;
+
+    JudoModelLoader modelHolder;
```

Comment:
Holder or Loader?

Reply:
Its a modelHolder, it loads one the model and hold it, so i thinks it is okey